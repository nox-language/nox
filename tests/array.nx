fun main() = (
    var len = 12;
    var a = [0; 12];
    var b = [7; 12];
    var value = a[0];
    printi(value);
    var value = b[0];
    printi(value);
    a[0] = 10;
    printi(a[0]);
    for i = 0 to len - 1 do
        a[i] = i;
    for i = 0 to len - 1 do
        printi(a[i]);
    var c = [
        40,
        41,
        42,
    ];
    printi(c[0]);
    printi(c[1]);
    printi(c[2]);
    var d = [12];
    printi(d[0]);
    var e = [1, 2];
    printi(e[0]);
    printi(e[1]);
)
