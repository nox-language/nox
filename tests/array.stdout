0
7
10
0
1
2
3
4
5
6
7
8
9
10
11
40
41
42
12
1
2
